datahub-graphql-core/src/main/resources/search.graphql

```diff
@@ -143,12 +147,44 @@ input FacetFilterInput {
   """
   Name of field to filter by
   """
-	field: String!
+  field: String!
+
+  """
+  Value of the field to filter by (soon to be deprecated)
+  """
+  value: String!
+
+  """
+  Values of the field to filter by
+  """
+  values: [String!]
 
-	"""
-	Value of the field to filter by
-	"""
-	value: String!
+  """
+  If the filter should or should not be matched
+  """
+  negated: Boolean
+
+  """
+  Condition for the values. How to If unset, assumed to be equality
+  """
+  condition: SearchCondition
```

Comment:
this pr reformats things, but this is the only actual change to this file ^ aside from formatting.

Reply:
Thanks for the fyi!